.gitignore

```diff
@@ -23,29 +21,21 @@ DerivedData
 project.xcworkspace
 
 # Android/IntelliJ
-#
 build/
 .idea
 .gradle
 local.properties
 *.iml
 
 # node.js
-#
 node_modules/
 npm-debug.log
 yarn-error.log
 .eslintcache
 
-# BUCK
-buck-out/
-\.buckd/
-android/app/libs
+# Android Signing Keys
```

Comment:
looking at this again, this section could move down and be joined up with the `# android keystores` section

Reply:
Actually, the `*.keystore` rule is a duplicate.